euth_wagtail/templates/account/signup.html

```diff
@@ -1,22 +1,24 @@
 {% extends 'base.html' %}
 
-{% load static widget_tweaks socialaccount i18n%}
+{% load static widget_tweaks socialaccount i18n %}
 
 {% block title %}{% trans 'Register an account' %}{% endblock %}
 
 {% block content %}
 <div class="account__container">
     <div class="account__container--left">
-        <div class="skinny-form">
-            <header class="skinny-form-header">
-                <ul class="nav nav-tabs" role="tablist">
-                    <li role="presentation"><a href="{% url 'account_login' %}?next={{redirect_field_value|urlencode}}">{% trans "login" %}</a></li>
-                    <li role="presentation" class="active"><a href="{% url 'account_signup' %}?next={{redirect_field_value|urlencode}}">{% trans "register" %}</a></li>
-                </ul>
+        <div class="general-form">
+            <header class="form-inner">
+                <h1>{% trans "register" %}</h1>
+                <p>
+                    {% trans "I already have an account, " %}
+                    <a href="{% url 'account_login' %}?next={{redirect_field_value|urlencode}}">
+                        {% trans "login" %}
+                    </a>
+                </p>
             </header>
-
             {% block form %}
-            <form class="skinny-form-inner" role="form" name="RegisterForm" action="{% url 'account_signup' %}" method="post" >
+            <form class="form-inner" role="form" name="RegisterForm" action="{% url 'account_signup' %}" method="post" >
```

Comment:
the header and form are both using form-inner which is creating too much padding between header and form